programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -136,67 +136,31 @@ pub fn set_cross_domain_admin(ctx: Context<SetCrossDomainAdmin>, cross_domain_ad
     Ok(())
 }
 
-#[event_cpi]
 #[derive(Accounts)]
-#[instruction(origin_token: Pubkey, destination_chain_id: u64)]
-pub struct SetEnableRoute<'info> {
-    #[account(constraint = is_local_or_remote_owner(&signer, &state) @ SvmError::NotOwner)]
-    pub signer: Signer<'info>,
-
+pub struct CreateVault<'info> {
```

Comment:
I'm a bit confused by this change. Was the point of this PR not to just remove enable deposit route check?

Reply:
yes, alternative would be to remove SetEnableRoute altogether and let the operator create vault PDA manually.

Reply:
I did end up deleting the instruction altogether in the last commit, since vault ATA can be created by anyone directly via associated token program.